app/views/reminder_mailer/update_user_reminder_email.html.haml

```diff
@@ -0,0 +1,18 @@
+%h2 Ihr Skills-Profil wartet auf Sie
+
+%p
+  Hallo,
+%p
+  Ihr Skills-Profil wurde seit über einer langen Zeit nicht mehr aktualisiert. Eine aktuelle Übersicht Ihrer Fähigkeiten hilft uns, passende und spannende Projekte für Sie zu finden.
+%p
+  Nehmen Sie sich einen Moment Zeit, um Ihr Profil auf den neuesten Stand zu bringen.
+
+%a.button{href: "https://skills.puzzle.ch"} Profil jetzt aktualisieren
```

Comment:
Have you considered that a hardcoded URL isn't that smart? :wink: 
Please figure out a solution to make this dynamically.